os/board/rtl8721csm/src/component/common/api/wifi_interactive_mode.c

```diff
@@ -445,7 +445,7 @@ int8_t cmd_wifi_connect(trwifi_ap_config_s *ap_connect_config, void *arg)
 		password = ap_connect_config->passphrase;
 		ssid_len = strlen((const char *)ssid);
 		password_len = ap_connect_config->passphrase_length;
-		key_id = 1; //Foucus
+		key_id = 0;

```

Comment:
I agreed Sangam's opinion. And in this function(`cmd_wifi_connect()`), all `key_id` are used as 0, so it is better to initialize at the first of the function.

Reply:
I agree with you, thanks